podcasts/PlaybackManager.swift

```diff
@@ -1731,7 +1735,17 @@ class PlaybackManager: ServerPlaybackDelegate {
                     if Settings.legacyBluetoothModeEnabled(), seekEvent.positionTime < 1 {
                         FileLog.shared.addMessage("Remote control: ignoring changePlaybackPositionCommand, it's to 0 and legacy bluetooth mode is on")
                     } else {
-                        FileLog.shared.addMessage("Remote control: changePlaybackPositionCommand")
+                        FileLog.shared.addMessage("Remote control: changePlaybackPositionCommand to \(seekEvent.positionTime)")
+
+                        if FeatureFlag.limitPlaybackPositionChanges.enabled {
+                            // Check if we're still in the limiting window after an episode change
+                            if let switchTime = episodeSwitchTime,
+                               Date().timeIntervalSince(switchTime) < 2.0 {
```

Comment:
Just a small nit: Can we switch `Date()` to `Date.now` just find it a bit more explicit that is current date.